src/test/kotlin/no/skatteetaten/aurora/gobo/GoboTestMain.kt

```diff
@@ -0,0 +1,91 @@
+package no.skatteetaten.aurora.gobo
+
+import com.fkorotkov.kubernetes.metadata
+import com.fkorotkov.kubernetes.newPod
+import io.kubernetes.client.PortForward
+import io.kubernetes.client.openapi.Configuration
+import io.kubernetes.client.util.Config
+import io.kubernetes.client.util.Streams
+import mu.KotlinLogging
+import no.skatteetaten.aurora.gobo.graphql.PROFILE_LOCAL_OCP04
+import no.skatteetaten.aurora.kubernetes.KubernetesReactorClient
+import no.skatteetaten.aurora.kubernetes.config.kubernetesToken
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+import org.springframework.context.annotation.Profile
+import org.springframework.stereotype.Component
+import java.net.ServerSocket
+import javax.annotation.PostConstruct
+import kotlin.concurrent.thread
+
+private val logger = KotlinLogging.logger {}
+
+data class PortForwardInput(val namespace: String, val application: String, val localPort: Int)
+
+@Profile(PROFILE_LOCAL_OCP04)
+@Component
+class MokeyPortForward(val kubernetesClient: KubernetesReactorClient) {
+
+    private val targetPort = 8080
+
+    @PostConstruct
+    fun init() {
+        Configuration.setDefaultApiClient(Config.defaultClient())
+        listOf(
+            PortForwardInput("aup", "mokey", 9999),
+            PortForwardInput("aup", "dbh", 9998)
+        ).forEach {
+            portForward(it)
+        }
+    }
+
+    fun portForward(input: PortForwardInput) {
+        val pod = kubernetesClient.getMany(
+            resource =
+            newPod {
+                metadata {
+                    namespace = input.namespace
+                    labels = mapOf("app" to input.application)
```

Comment:
Det er en liten mulighet for feil hvis det skjer samtidig med rolling deploy? Men kan sikkert leve med det.